File: task-03/prime_number.c
#include <stdio.h>

int main() {
    int n, i, j;

    printf("Enter a number (n): ");
    scanf("%d", &n);
    
    if (n<2) {
        printf("There are no prime numbers below 2");
    } else {
    printf("Prime numbers up to %d:\n", n);
        
    }

    for (i = 2; i <= n; i++) {
        int isPrime = 1;

        for (j = 2; j < i; j++) {
            if (i % j == 0) {
                isPrime = 0;
                break;
            }
        }

        if (isPrime) {
            printf("%d ", i);
        }
    }

    printf("\n");

    return 0;
}

File: task-03/prime_number.go
package main

import "fmt"

func main() {
    var n int
    fmt.Print("Enter a number (n): ")
    fmt.Scan(&n)
    
    if (n<2) {
        fmt.Print("There are no prime numbers below 2")
    } else {
        fmt.Printf("Prime numbers up to %d: ", n)
    }

    for i := 2; i <= n; i++ {
        isPrime := true
        for j := 2; j*j <= i; j++ {
            if i%j == 0 {
                isPrime = false
                break
            }
        }
        if isPrime {
            fmt.Printf("%d ", i)
        }
    }

    fmt.Println()
}

